odl/discr/lp_discr.py

```diff
@@ -34,9 +34,7 @@
 from odl.discr.partition import RectPartition, uniform_partition_fromintv
 from odl.set.sets import RealNumbers, ComplexNumbers
 from odl.set.domain import IntervalProd
-from odl.space.cu_ntuples import CUDA_AVAILABLE, CudaFn
-from odl.space.fspace import FunctionSpace
-from odl.space.ntuples import Fn
+from odl.space import FunctionSpace, FN_IMPLS
```

Comment:
Probably if you search for "SUPPORTED" you'll get some hits. We also have ASTRA, STIR, ...